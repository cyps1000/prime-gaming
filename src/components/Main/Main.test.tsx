/**
 * @see https://testing-library.com/docs/react-testing-library/intro
 * @see https://www.robinwieruch.de/react-testing-library
 */
import { render } from "@testing-library/react";

/**
 * Mocking the useTranslation hook
 * @see https://github.com/i18next/react-i18next/issues/876
 *
 */
jest.mock("react-i18next", () => ({
  useTranslation: () => ({ t: (key: string) => key }),
}));

/**
 * Default test
 */
describe("Main", () => {
  it("renders the component", () => {
    // render(<Main />);
  });
});
