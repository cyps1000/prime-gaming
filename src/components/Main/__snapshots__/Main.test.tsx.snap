// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Main Snapshot Test passes the snapshot test 1`] = `
"<div class=\\"makeStyles-root-5\\">
  <h1>Main</h1>
</div>"
`;
